tigerpath/majors_and_certificates/majors/CLA_2018.json

```
@@ -0,0 +1,518 @@
+              "req_list": [
+                {
+                  "name": "Original Languages",
+                  "max_counted": null,
+                  "min_needed": 5,
+                  "explanation": "Of these, five must be in the original languages at the 200 level or above, including at least one course at the 300 level. The combination of LAT 104-108, or CLG/LAT 105-108, may be counted as the equivalent of one 200-level course.",
+                  "req_list": [
+                    {
+                      "name": null,
+                      "max_counted": 4,
+                      "min_needed": null,
+                      "explanation": null,
+                      "req_list": [
+                        {
+                          "name": null,
+                          "max_counted": null,
+                          "min_needed": null,
+                          "explanation": null,
+                          "course_list": [
+                            "LAT 2*",
+                            "CLG 2*"
+                          ]
+                        },
+                        {
+                          "name": null,
+                          "max_counted": 1,
+                          "min_needed": "ALL",
+                          "explanation": null,
+                          "course_list": [
+                            "LAT 105",
+                            "LAT 108"
+                          ]
+                        },
+                        {
+                          "name": null,
+                          "max_counted": 1,
+                          "min_needed": "ALL",
+                          "explanation": null,
+                          "course_list": [
+                            "CLG 105",
+                            "CLG 108"
+                          ]
+                        }
+                      ]
+                    },
+                    {
+                      "name": null,
+                      "max_counted": null,
+                      "min_needed": 1,
+                      "explanation": null,
+                      "course_list": [
+                        "CLG 3*",
+                        "LAT 3*",
+                        "CLG 4*",
+                        "LAT 4*",
+                        "CLG 5*",
+                        "LAT 5*"
+                      ]
+                    }
+                  ]
+                },
+                {
+                  "name": "Ancient History",
+                  "max_counted": null,

```

And this should be limited to 3, so that it doesn't replace an "Original Languages" course.
```suggestion
                  "max_counted": 3,
```